### PR TITLE
Fix libtar integration

### PR DESCRIPTION
**Pull request by barci2**

### CODE REVIEW

File: build_scripts/build_deps.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ echo ${EXTRA_PROTOBUF_FLAGS}
 echo ${OPENCV_TOOLCHAIN_FILE}
 echo ${EXTRA_FLAC_FLAGS}
 echo ${EXTRA_LIBSND_FLAGS}
+echo ${EXTRA_LIBTAR_FLAGS}
```
Comment (JanuszL) [inline build_scripts/build_deps.sh:37]: I don't think we need that here. Do you know about any, platform specific flags that we need to pass to libtar?

Comment (barci2) [reply build_scripts/build_deps.sh:37]: Nope, just added it because it's also for libsnd. Should I remove it?

Comment (JanuszL) [reply build_scripts/build_deps.sh:37]: Yes, please. It is not needed here.

Comment (barci2) [reply build_scripts/build_deps.sh]: done

**Review by JanuszL — APPROVED** [review]

**Pull request merged**